wp_api/src/lib.rs

```diff
@@ -218,3 +206,31 @@ macro_rules! generate {
 }
 
 uniffi::setup_scaffolding!();
+
+#[cfg(test)]
+mod lib_tests {
```

Comment:
When making this mod as `tests` like we usually do in other source files, running this mod in VS code would run the tests in the whole library. Maybe the "tests" mod at the root level is reserved by cargo or something.

Reply:
So, I took a look what VS code does. Below is the command VS code runs when click "Run tests" button under a `mod test_mod_name`:

```
cargo test --package wp_api --lib --all-features -- <test_mod_name> --show-output 
```

When naming this module as `tests`, VS code would run `cargo test [...] -- tests`. Since pretty much every test function would have `tests` in their qualified name, all tests are executed.